CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
+## 0.6.1
```

Comment:
is this require a version bump?

Reply:
Yes. This PR contains add package-lock.json. This is another problem about tslint.